packages/kokoas-client/src/pages/orderInvoice/sections/result/ResultRow.tsx

```diff
@@ -3,10 +3,16 @@ import { RowLayout, RowLayoutProps } from './RowLayout';
 
 
 export const ResultRow = (props: RowLayoutProps) => {
+  const { 
+    orderAmount, 
+    paymentAmount,
+  } = props;
 
   return (
     <RowLayout 
       {...props}
+      orderAmount={orderAmount?.toLocaleString()}
+      paymentAmount={paymentAmount?.toLocaleString()}
```

Comment:
```suggestion
      orderAmount={`${orderAmount?.toLocaleString()}円`}
      paymentAmount={`${paymentAmount?.toLocaleString()}円`}
```
契約一覧等と合わせて、"円"があった方が良いかなと思いました。

Reply:
そうですね。ありがとうございます！そのまま実装しました。